models/brats_mri_segmentation/docs/README.md

```diff
@@ -8,6 +8,8 @@ The model is trained to segment 3 nested subregions of primary brain tumors (gli
 - The TC describes the bulk of the tumor, which is what is typically resected. The TC entails the ET, as well as the necrotic (fluid-filled) and the non-enhancing (solid) parts of the tumor.
 -  The WT describes the complete extent of the disease, as it entails the TC and the peritumoral edema (ED), which is typically depicted by hyper-intense signal in FLAIR.
 
+![](https://developer.download.nvidia.com/assets/Clara/Images/clara_pt_brain_mri_segmentation_workflow.png)
+
```

Comment:
Thanks @myron . Please also update the changelog and version number in metadata.json
and put new weights download link into large_files.yml

Reply:
@yiheng-wang-nv  I've updated the changelog and version number.  based on the conversation now, we decided to keep the existing weights as is. please review and merege